cumulusci.yml

```diff
@@ -587,6 +587,13 @@ tasks:
                 errSettings.%%%NAMESPACE%%%OverrideFeature_PilotEnabled__c = true;
                 upsert errSettings;
 
+    deploy_core_mock_class:
+        description: Deploy a mock apex class that will prevent compilation errors when the packaged code references file-based apex in the core stack for instrumentation purposes
+        class_path: cumulusci.tasks.salesforce.Deploy
+        group: Salesforce Metadata
+        options:
+            path: unpackaged/pre/core_apex_class_mock
```

Comment:
@davidmreed Out of curiosity, am I able to update the `Codeowners` file to add an entry for `unpackaged/pre/core_apex_class_mock` or is that only for RE related folders?

Reply:
You can do that. I'll probably have to approve the change (which I'm happy to do), but it's a perfectly legitimate use of CODEOWNERS.